src/Version/Version.cpp

```diff
@@ -67,6 +75,33 @@ std::string getOnnxMlirFullRepositoryVersion(bool toIncludeLLVM) {
   return buf;
 }
 
+std::string getProductFullVersion() {
+  std::string buf;
+  llvm::raw_string_ostream os(buf);
+#if defined(ONNX_MLIR_VENDOR)
+  os << ONNX_MLIR_VENDOR << " " << PRODUCT_VERSION_MAJOR << '.';
+  os << PRODUCT_VERSION_MINOR << '.';
+  os << PRODUCT_VERSION_PATCH << '-' << PRODUCT_ID;
+#endif
+  return buf;
+}
+
+std::string getOnnxMlirCommitVersion() {
+  std::string buf;
+  llvm::raw_string_ostream os(buf);
+  std::string OnnxMlirPath = getOnnxMlirRepositoryPath();
+  std::string OnnxMlirCommit = getOnnxMlirCommit();
```

Comment:
These aren't classes so shouldn't the names be `onnxMlirPath` and `onnxMlirCommit` (ie lowercase o at the start)?

Reply:
Updated!